sql/hive/src/test/scala/org/apache/spark/sql/hive/execution/HiveUDAFSuite.scala

```diff
@@ -149,6 +149,14 @@ class HiveUDAFSuite extends QueryTest with TestHiveSingleton with SQLTestUtils {
       }
     }
   }
+
+  test("HiveUDAF with 0 rows throws NPE") {
+    sql("create table abc(a int)")
+    sql("insert into abc values (1)")
+    sql("insert into abc values (2)")
+    sql("select histogram_numeric(a,2) from abc").show
```

Comment:
Are these two lines above needed for the test?

Reply:
Just to show the effect only when query return 0 rows, i am ok to drop it, they are not nessessary

Reply:
rewritten